piff/basis_interp.py

```diff
@@ -249,26 +292,46 @@ def _solve_direct(self, stars, logger):
 
         # Build ATA and ATb by accumulating the chunks for each star as we go.
         nq = np.prod(self.q.shape)
+        logger.info(f'nq = {nq}')
```

Comment:
This is probably a bit baroque for an info.  `logger.debug` would be better.

Reply:
I forgot to remove this line, it was for my personal use when i was debugging. I removed it.